python/cuml/linear_model/logistic_regression_mg.pyx

```diff
@@ -82,8 +82,28 @@ cdef extern from "cuml/linear_model/qn_mg.hpp" namespace "ML::GLM::opg" nogil:
 
 class LogisticRegressionMG(MGFitMixin, LogisticRegression):
 
-    def __init__(self, *, handle=None):
-        super().__init__(handle=handle)
+    def __init__(
+        self,
```

Comment:
In other dask estimators, we pass `**kwargs` through to the super class so that we don't need to specify the defaults twice in the cython layer. Is there a reason we can't do that here? As an [example](https://github.com/rapidsai/cuml/blob/e31095763b31c14bb5c1b318f4cd2806ac9633b3/python/cuml/neighbors/kneighbors_classifier_mg.pyx#L68).